jest.config.ts

```diff
@@ -16,8 +16,13 @@ export default {
   preset: 'ts-jest',
   testEnvironment: 'jest-environment-node-single-context',
   setupFiles: ['dotenv/config'],
-  //setupFilesAfterEnv: ['<rootDir>src/setupTests.ts'],
   moduleFileExtensions: ['js', 'ts'],
+  testMatch: ['<rootDir>/src/**/?(*.)+(spec|test).[jt]s?(x)'],
```

Comment:
I am not sure about this. All the other repo are using the all tests in the tests folder pattern (except snapshot vue app)

Reply:
what is your concern? I have added this line because I think better to place all unit test files near the files they test.

Reply:
The concern is that all the other repo in the organization are following the structure where all units test are located in test/units instead of being next to the file being tested.